typescript/sdk/src/utils/ism.ts

```diff
@@ -11,3 +13,61 @@ export function multisigIsmVerificationCost(m: number, n: number): number {
   // @ts-ignore
   return multisigIsmVerifyCosts[`${n}`][`${m}`];
 }
+
+// Function to recursively remove 'address' properties and lowercase string properties
+export function normalizeConfig(obj: WithAddress<any>): any {
+  return sortValidatorsAndModulesInConfig(loweCaseConfig(obj));
+}
+
+function loweCaseConfig(obj: any): any {
```

Comment:
and also fix where it's called
```suggestion
function lowerCaseConfig(obj: any): any {
```